src/Initializer/LTS.h

```diff
@@ -138,8 +138,8 @@ struct seissol::initializers::LTS {
     tree.addBucket(displacementsBuffer,                         PAGESIZE_HEAP,      MEMKIND_TIMEDOFS );
 
 #ifdef ACL_DEVICE
-    tree.addVar(   localIntegrationOnDevice,   LayerMask(Ghost),  1,      seissol::memory::DeviceGlobalMemory );
-    tree.addVar(   neighIntegrationOnDevice,   LayerMask(Ghost),  1,      seissol::memory::DeviceGlobalMemory );
+    tree.addVar(   localIntegrationOnDevice,           LayerMask(Ghost),  1,      seissol::memory::DeviceGlobalMemory );
```

Comment:
Can you remove these changes? They clutter the git history and they are no functional changes.

Reply:
yes, sure